sql-schema/127.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS `ipmi_sensors`;
```

Comment:
How is this usable though for others though? It only contains HP hardware and just seems like another list of things to keep upto date.

We have 'components' which you could make use of to add things to the DB rather than creating a new 'limited use' table.